lib/slide_hub/convert_util.rb

```diff
@@ -87,5 +95,21 @@ def get_local_file_list(dir, extension)
       end
       list.sort
     end
+
+    private
+
+      def exec_command(cmd)
+        logger.info(cmd)
+        Open3.popen3(cmd) do |_i, o, e, w|
+          out = o.read
+          err = e.read
+          logger.info out if out.length > 0
+          logger.error err if err.length > 0
```

Comment:
__[RuboCop]__

Style/ZeroLengthPredicate: Use `!empty?` instead of `length > 0`.


<!-- Comment by [Sider](https://sider.review) -->